src/picongpu/include/plugins/radiation/Radiation.hpp

```diff
@@ -606,6 +610,7 @@ class Radiation : public ISimulationPlugin
       splash::DataCollector::initFileCreationAttr(fAttr);
 
       fAttr.fileAccType = splash::DataCollector::FAT_READ;
+      fAttr.enableCompression = compressionOn;
```

Comment:
what do you mean with that? he just added an option for that, if he does not provide an option he has to decide on "always on/always off".